flytekit/models/common.py

```diff
@@ -60,9 +86,9 @@ def short_string(self):
         """
         :rtype: Text
         """
-        literal_str = re.sub(r"\s+", " ", str(self.to_flyte_idl())).strip()
+        str_repr = _repr_idl_yaml_like(self.to_flyte_idl(), indent=2).rstrip(os.linesep)
         type_str = type(self).__name__
-        return f"[Flyte Serialized object: Type: <{type_str}> Value: <{literal_str}>]"
+        return f"Flyte Serialized object ({type_str}):" + os.linesep + str_repr
```

Comment:
Can we add a few more tests to cover the different types of literals (collections, maps, etc)? These are useful especially in flyteremote when debugging.

Reply:
I added more tests in https://github.com/flyteorg/flytekit/pull/2647/commits/21372e4748801a2baf68b8b46e31f01a0875ad56